msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsUser.jsp

```diff
@@ -189,9 +189,10 @@
 	  	<h:outputText rendered="#{ForumTool.anonymousEnabled && ForumTool.siteHasAnonymousTopics && !mfStatisticsBean.pureAnon}" value="#{msgs.stat_forum_anonymous_omitted}" styleClass="instruction" />
 
         <div>
-            <h:outputFormat value="#{msgs.stat_percent_read_user}" escape="false">
-                <f:param value="#{mfStatisticsBean.getPercentRead()}" />
-            </h:outputFormat>
+            <h:outputText value="#{msgs.stat_percent_read_user}" escape="false" />
+            <h:outputText value="#{mfStatisticsBean.getPercentRead()}">
+                <f:convertNumber type="percent" />
```

Comment:
so what's the default number of digits here? maxFractionDigits ?  should be no more than 2 i hope

Reply:
In my testing it rounds to whole numbers: 2/3 read posts displays as 67%, 1/3 displays as 33%, etc.